broker/src/main/java/org/apache/rocketmq/broker/subscription/SubscriptionGroupManager.java

```diff
@@ -121,12 +121,12 @@ public SubscriptionGroupConfig findSubscriptionGroupConfig(final String group) {
         SubscriptionGroupConfig subscriptionGroupConfig = this.subscriptionGroupTable.get(group);
         if (null == subscriptionGroupConfig) {
             if (brokerController.getBrokerConfig().isAutoCreateSubscriptionGroup() || MixAll.isSysConsumerGroup(group)) {
-                subscriptionGroupConfig = new SubscriptionGroupConfig();
-                subscriptionGroupConfig.setGroupName(group);
-                SubscriptionGroupConfig preConfig = this.subscriptionGroupTable.putIfAbsent(group, subscriptionGroupConfig);
-                if (null == preConfig) {
-                    log.info("auto create a subscription group, {}", subscriptionGroupConfig.toString());
-                }
+                subscriptionGroupConfig = this.subscriptionGroupTable.computeIfAbsent(group, (key) -> {
```

Comment:
The function is only called when the value is absent. So this can avoid unnecessary object creation.